llvm/include/llvm/Transforms/Utils/LoopMetadata.h

```diff
@@ -0,0 +1,73 @@
+//===-- LoopMetadata.h - Convert Loop Metadata to assumptions --*- C++ -*-===//
```

Comment:
That name is very generic, how about `LoopIterCountsToAssumptions` or `LoopMetadataToAssumptions`?